analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ExecutingSqlQueriesBase.cs

```diff
@@ -133,7 +127,7 @@ protected override void Initialize(TrackerInput input)
 
             var pa = Language.Tracker.PropertyAccess;
             pa.Track(input,
-                pa.MatchProperty(properties),
+                pa.MatchProperty(checkOverridenProperties: true, properties),
```

Comment:
Not needed, no confusion around
```suggestion
                pa.MatchProperty(true, properties),
```